docs/markdown/snippets/add_release_note_snippets_here

```diff
@@ -1,3 +1,6 @@
 ## Added `cpp_std` option for the Visual Studio C++ compiler
 Allows the use of C++17 features and experimental not-yet-standardized
 features. Valid options are `c++11`, `c++14`, `c++17`, and `c++latest`.
+## Added c_std values 'c17' and 'c18' for recent GCC and Clang versions
+For gcc >=8.0.0, allow c_std='c17' 'c18' 'gnu17' 'gnu18'.
+For clang >=10.0.0 on OSX and >=7.0.0 otherwise, allow c_std='c17' 'gnu17'.
```

Comment:
This should go in a separate file in`docs/markdown/snippets`, for example `docs/markdown/snippets/add_c17_c18_standards.md`

Reply:
Done.

Reply:
A separate *.md docs is now in the snippets directory. I'll mark this resolved.